app.rb

```diff
@@ -2,26 +2,28 @@
 require './student'
 require './book'
 require './rental'
+require './storage'
 
 class App
   attr_accessor :persons, :books, :rentals
 
   def initialize
+    @storage = Storage.new('storage')
     @persons = []
     @books = []
     @rentals = []
   end
 
-  def get_type(person)
-    person&.specialization.nil? ? '[Student]' : '[Teacher]'
+  def get_type(_person)
+    @person&.specialization.nil? ? '[Student]' : '[Teacher]'
   end
 
   def list_persons(display_num: false)
     # if there are no persons
-    return puts "\nERROR:\nThere are no persons created yet!\n\n" if persons.nil? || persons.empty?
+    return puts "\nERROR:\nThere are no persons created yet!\n\n" if @persons.nil? || @persons.empty?
```

Comment:
- Great job is done handling the case when one or more of the files are missing to avoid the app from throwing errors 🥇